README.md

```diff
@@ -56,3 +57,4 @@ This value should be `true` if AWS S3 is used as storage.
 | DATA_EXPORT_JOB_UPDATE_TOPIC_PARTITIONS     | 50                     | Number of partitions for topic                                                                                                                                                                                                     |
 | KAFKA_CONCURRENCY_LEVEL                     | 30                     | Concurrency level of kafka listener                                                                                                                                                                                                |
 | LOCAL_FS_COMPOSE_WITH_AWS_SDK               | false                  | Specify if AWS S3 is used as local files storage                                                                                                                                                                                   |
+| E_HOLDINGS_BATCH_JOB_CHUNK_SIZE             | 100                    | Specify chunk size for eHoldings export job which will be used to query data from kb-ebsco, write to database, read from database and write to file                                                                                |
```

Comment:
The maximum amount records that could be loaded from kb-ebsco is 100. So, we should have some limit for this env var.

Reply:
Good catch. Moved into separate variable, added limitation